assets/app/scripts/controllers/deployment.js

```diff
@@ -104,6 +104,8 @@ angular.module('openshiftConsole')
                 };
               }
               $scope.deployment = deployment;
+              $scope.logOptions.container = $filter("annotation")(deployment, "pod");
+              $scope.logCanRun = !(_.includes(['New', 'Pending'], $filter('deploymentStatus')(deployment)));
```

Comment:
The updates are complicated enough I'd rather pull them into a function instead of repeating them twice (here and the other controllers).